pyscriptjs/tests/unit/main.test.ts

```diff
@@ -1,69 +1,73 @@
-import { describe, it, beforeEach, expect } from '@jest/globals';
-import { UserError, ErrorCode } from '../../src/exceptions';
-import { PyScriptApp } from '../../src/main';
+import { describe, it, expect } from '@jest/globals';
 
-describe('Test withUserErrorHandler', () => {
-    class MyApp extends PyScriptApp {
-        myRealMain: any;
+describe('Placeholder', () => {
+    it('this is a placeholder, we need to fix and re-enable the following tests', () => {
+        expect(true).toBe(true);
+    });
+});
 
-        constructor(myRealMain) {
-            super();
-            this.myRealMain = myRealMain;
-        }
+// describe('Test withUserErrorHandler', () => {
+//     class MyApp extends PyScriptApp {
+//         myRealMain: any;
```

Comment:
don't use `//` comments here. It's preferable to use `/* */` for the whole file: this way:
1. the diff is shorter / easier to review
2. git blame continues to work and will show the original author of the line, even after we remove the comment

Reply:
thanks, done in https://github.com/pyscript/pyscript/pull/1258/commits/4b124c3d65c53bc846605b6fa673efa5d4715648